Tests/Pcap++Test/Tests/LiveDeviceTests.cpp

```diff
@@ -65,6 +65,31 @@ static bool packetArrivesBlockingModeStartCapture(pcpp::RawPacket* rawPacket, pc
 	return false;
 }
 
+static bool packetArrivesBlockingModeStartCaptureUsingLambda(pcpp::RawPacket* rawPacket, pcpp::PcapLiveDevice* dev, void* userCookie)
```

Comment:
There is no test that uses this static method, so it doesn't really run anywhere

Reply:
ops. true. I always forget to put the new test into the list.